src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static isAnimalDead(animal) {
+    if (animal.health <= 0) {
+      Animal.alive = Animal.alive.filter(
+        (currentAnimal) => currentAnimal !== animal,
+      );
+    }
```

Comment:
It don't have to be static. We can call this method on object instance.` this.isAnimalDead`